rs/nns/governance/src/neuron/types.rs

```diff
@@ -1134,9 +1136,14 @@ impl Neuron {
     }
 }
 
-impl From<Neuron> for NeuronProto {
-    fn from(neuron: Neuron) -> Self {
-        let visibility = neuron.visibility().map(|visibility| visibility as i32);
+impl Neuron {
+    pub fn into_proto(self, now_seconds: u64) -> NeuronProto {
```

Comment:
Can't hang onto the old interface, because conversion requires a new piece of auxiliary data (i.e. the current time).

Thus, instead of

```
NeuronProto::from(native_neuron)
```

you have to instead do (something like)

```
native_neuron.into_proto(timestamp)
```

😇